Application Lifecycle Management/instances-api.md

```diff
@@ -1497,89 +1790,304 @@ GET /services/instances/{instance_id}/activity?operation=deploy
 | created | string | Activity creation date. |
 |* event | string | Activity event. |
 
+
```

Comment:
Add this line with what * means. By example:
* only showed when a script executes